src/domain/permissions.js

```diff
@@ -65,9 +61,7 @@ export default {
   MANAGE_IMPORT,
   MANAGE_EXPORT,
   DELETE_CDN_FILE,
-  CREATE_ROOMS,
+  OWN_ROOMS,
```

Comment:
When you rename a constant, could you please do a full text search and rename everywhere. You missed one!